### PR TITLE
Update external links in security guide [ci-skip]

### PR DESCRIPTION
**Pull request by djpowers**
This changes fixes broken links, and updates URLs to remove redirects in order to preserve longevity.

### CODE REVIEW

File: guides/source/security.md [hunk 1]
```diff
@@ -787,7 +787,7 @@ This example pops up a message box. It will be recognized by the above `sanitize
 
 _In order to understand today's attacks on web applications, it's best to take a look at some real-world attack vectors._
 
-The following is an excerpt from the [Js.Yamanner@m](http://www.symantec.com/security_response/writeup.jsp?docid=2006-061211-4111-99&tabid=1) Yahoo! Mail [worm](http://groovin.net/stuff/yammer.txt). It appeared on June 11, 2006 and was the first webmail interface worm:
+The following is an excerpt from the [Js.Yamanner@m](http://securityresponse.symantec.com/avcenter/venc/data/js.yamanner@m.html) Yahoo! Mail [worm](https://community.broadcom.com/symantecenterprise/communities/community-home/librarydocuments/viewdocument?DocumentKey=12d8d106-1137-4d7c-8bb4-3ea1faec83fa). It appeared on June 11, 2006 and was the first webmail interface worm:
```
Comment (jonathanhefner) [inline guides/source/security.md:790]: > http://securityresponse.symantec.com/avcenter/venc/data/js.yamanner@m.html

This URL appears to redirect to https://www.broadcom.com/support/security-center. :thinking:

Comment (djpowers) [reply guides/source/security.md:790]: Ah, not sure if it might make sense to just remove the link altogether on that one then.

Comment (jonathanhefner) [reply guides/source/security.md:790]: Yes, perhaps just merge the text into the next link:

```suggestion
The following is an excerpt from the [Js.Yamanner@m Yahoo! Mail worm](https://community.broadcom.com/symantecenterprise/communities/community-home/librarydocuments/viewdocument?DocumentKey=12d8d106-1137-4d7c-8bb4-3ea1faec83fa). It appeared on June 11, 2006 and was the first webmail interface worm:
```

**Comment by jonathanhefner** [comment]
@djpowers Would you mind also [squashing the commits](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits)?  Then I'll give this a merge!

**Comment by djpowers** [comment]
@jonathanhefner, thanks, should be all set!

File: guides/source/security.md [hunk 2]
```diff
@@ -689,7 +689,7 @@ The most common entry points are message posts, user comments, and guest books,
...
+During the second half of 2007, there were 88 vulnerabilities reported in Mozilla browsers, 22 in Safari, 18 in IE, and 12 in Opera. The Symantec Global Internet Security threat report also documented 239 browser plug-in vulnerabilities in the last six months of 2007. [Mpack](https://www.pandasecurity.com/en/mediacenter/malware/mpack-uncovered/) is a very active and up-to-date attack framework which exploits these vulnerabilities. For criminal hackers, it is very attractive to exploit an SQL-Injection vulnerability in a web application framework and insert malicious code in every textual table column. In April 2008 more than 510,000 sites were hacked like this, among them the British government, United Nations, and many more high profile targets.
```
Comment (marivaldo) [inline guides/source/security.md:692]:
```suggestion
During the second half of 2007, there were 88 vulnerabilities reported in Mozilla browsers, 22 in Safari, 18 in IE, and 12 in Opera. The Symantec Global Internet Security threat report also documented 239 browser plug-in vulnerabilities in the last six months of 2007. [Mpack](https://www.pandasecurity.com/en/mediacenter/malware/mpack-uncovered/) is a very active and up-to-date attack framework which exploits these vulnerabilities. For criminal hackers, it is very attractive to exploit a SQL-Injection vulnerability in a web application framework and insert malicious code in every textual table column. In April 2008 more than 510,000 sites were hacked like this, among them the British government, United Nations, and many more high profile targets.
```

**Review by marivaldo — CHANGES_REQUESTED** [review]

File: guides/source/security.md (hunk 2 above)
Comment (djpowers) [reply guides/source/security.md]: Updated other instances of "an SQL" to "a SQL," which does seem to be the more prevalent usage:
- https://trends.google.com/trends/explore?q=%22a%20sql%22,%22an%20sql%22
- https://books.google.com/ngrams/graph?content=a+SQL%2Can+SQL&case_insensitive=on&year_start=1980&year_end=2008&corpus=15&smoothing=1

**Review by marivaldo — APPROVED** [review]

**Comment by jonathanhefner** [comment]
Thank you, @djpowers!

**Pull request merged**